Transparent image and removed dubble image

jeroentjuuhh opened the pull request:
Two small changes:
- Made the background image of the intro transparent.
- Removed a image that was there twice

TimvdLippe commented on style.css:
```diff
@@ -98,12 +100,25 @@ button:active {
-  background: url("images-optimized/cheers.png") no-repeat center;
-  background-size: 50%;
+  /* background: url("images-optimized/cheers.png") no-repeat center; */
+  /* background-size: 50%; */
```
Do not leave commented code

TimvdLippe commented on style.css:
```diff
@@ -55,6 +55,8 @@ section:nth-child(2n) {
 }
 
+
+
```
Unnecessary whitespace

TimvdLippe commented on index.html:
```diff
@@ -24,7 +24,10 @@
+        <h1 class="cheers">CHeers!</h1>
+        <br>
+        <br>
+        <br>
```
Do not use `<br>` for spacing (you should almost never use `<br>`). Instead, set the height in the CSS.

TimvdLippe commented on style.css:
```diff
@@ -133,14 +148,23 @@ button:active {
-  font-size: 4vw;
-  margin-left: 50px;
+  .leaderboard ol {
+    font-size: 4vw !important; 
```
Do not use `!important`. It should not be necessary to add this. Moreover, it negates the selector as specified on line 170, which would break 4k screens.

jeroentjuuhh commented:
Changed the requested changes.
### Requested changes:
- Removed unnecessary white spaces and commented code
- Removed the !important and created another solution for that.
- Implemented another solution to the <br> solution.

### Also done: 
- Changed the leaderboard names.

Hope these changes are correct!
@TimvdLippe

TimvdLippe commented on index.html:
```diff
@@ -25,10 +25,7 @@
-        <br>
-        <br>
-        <h2> It has begun!</h2>
+        <h2 style="padding-top: 200px"> It has begun!</h2>
```
Put this styling in the `styles.css`

TimvdLippe commented:
@jeroentjuuhh Please check how I reformulated the PR description to improve the eventual commit message in https://github.com/WISVCH/dies-2018/commit/451f18964add56376b9efafcd8cdc857fbe0d59f
